test/gurka/test_route_incidents.cc

```diff
@@ -147,26 +146,26 @@ void check_incident_locations(const valhalla::Api& api,
 struct test_reader : public baldr::GraphReader {
   test_reader(const boost::property_tree::ptree& pt) : baldr::GraphReader(pt) {
     tile_extract_.reset(new baldr::GraphReader::tile_extract_t(pt));
+    enable_incidents_ = true;
```

Comment:
now that we have actual incident loading, we need to turn on the incident loading singleton in config. for this test we override the functionality of the incident loader by letting the test inject incidents directly so here we just turnon the flag in the graph reader so that it will sitll call the incident tile function